src/test/resources/mock_uploader_local_files/README.txt

```diff
@@ -0,0 +1,4 @@
+NOTE: This directory will be automatically populated and emptied with dummy/mock files during unit and IT testing of FTP
```

Comment:
Are they dummy files or mock files? Or (somehow) both?

Reply:
They are dummy files.  I will remove the "mock".

Reply:
Done